sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v2/query.sql

```diff
@@ -0,0 +1,42 @@
+WITH wnp_visits AS (
+  SELECT
+    date,
+    visit_identifier,
+    TRIM(page_path_level1, '/') AS locale,
+    page_level_2 AS version,
+    mozfun.norm.browser_version_info(page_level_2) AS version_info,
+    country,
+    CAST(bounces AS boolean) AS is_bounce
```

Comment:
nit: explicit > implicit
```suggestion
    bounces > 0 AS is_bounce
```